src/FINDOLOGIC/Export/Helpers/DataHelper.php

```diff
@@ -10,6 +10,14 @@ public function __construct($message = 'Empty values are not allowed!')
     }
 }
 
+class ObjectNotAllowedException extends \RuntimeException
```

Comment:
Better naming would be something like `UnsupportedTypeException` or whatever better name you can think of, as we should change the check to check for acceptable values instead of one unacceptable one.